shogun-lib/src/main/java/de/terrestris/shogun/lib/security/access/entity/BaseEntityPermissionEvaluator.java

```diff
@@ -75,6 +75,16 @@ public Class<E> getEntityClassName() {
     public boolean hasPermission(User user, E entity, PermissionType permission) {
         final String simpleClassName = entity.getClass().getSimpleName();
 
+        if (hasPublicPermission(entity)) {
```

Comment:
Public access should be granted to `READ` permissions only. This should be checked here.